src/blocks/Questions/__stories__/Questions.mdx

```diff
@@ -23,7 +23,7 @@ import * as QuestionsStories from './Questions.stories.tsx';
 
 - `title: string` — Question title (on the right)
 - `text: string` — Question text
-- `link?: Link` — Link below the text (see [Link](?id=components-links-and-buttons-link--default&viewMode=docs))
+- `link?: Link` — Link below the text (see [Link](?id=components-links-and-buttons-link--default))
```

Comment:
It's not working right now.

https://preview.gravity-ui.com/page-constructor/1168/?path=/docs/documentation-types--docs#link